### DISCUSSION TITLE
getCurrentAppName这个函数的一个疑问

### DISCUSSION THREAD

**Discussion by zjlhope** [jd-opensource/micro-app#1002] [Q&A]
看源码，利用getCurrentAppName拿到当前的appName来做元素隔离的，但这个函数只会记录一个子应用的，这样多个子应用同时渲染做的元素隔离是怎么做到的呢

**Comment by bailicangdu** [comment]
对元素的操作时会重置appName，用于标记活跃状态的子应用

**Reply by zjlhope** [reply]
get，大概了解了，谢谢
